test/ui-test/schemaIsSetTest.ts

```diff
@@ -0,0 +1,43 @@
+import { WebDriver, VSBrowser, Key, InputBox, TextEditor, StatusBar, By } from 'vscode-extension-tester';
+
+/**
+ * @author Zbynek Cervinka <zcervink@redhat.com>
+ */
+export function schemaIsSetTest(): void {
+  describe('Verify that the right JSON schema has been selected', () => {
+    it('The right JSON schema has been selected', async function () {
+      this.timeout(30000);
+
+      const driver: WebDriver = VSBrowser.instance.driver;
+      await driver.actions().sendKeys(Key.F1).perform();
+
+      let input = await InputBox.create();
+      await input.setText('>new file');
+      await input.confirm();
+      await input.confirm();
+
+      await driver.actions().sendKeys(Key.chord(TextEditor.ctlKey, 's')).perform();
+      input = await InputBox.create();
+      await input.setText('~/kustomization.yaml');
+      await input.confirm();
+
+      await delay(10000);
```

Comment:
I would not use hard wait. And also using and Error to show a test failure is not good practice, we have a fail method from mocha which is better to use. 
In any case, you can create a wait condition for a schema object and test if it is undefined or fail otherwise. Example of wait: https://github.com/redhat-developer/vscode-extension-tester/blob/master/sample-projects/helloworld-sample/src/ui-test/notifications-test.ts#L13

Reply:
@odockal Hard wait replaced with a proper wait condition and also the Error has been removed.